kuma/wiki/tests/test_templates.py

```diff
@@ -1329,3 +1329,110 @@ def _translation_data():
         'content': 'loremo ipsumo doloro sito ameto',
         'toc_depth': Revision.TOC_DEPTH_H4,
     }
+
+
+def test_zone_styles(client, doc_hierarchy_with_zones, root_doc):
+    """
+    Check document page for zone-style-related features.
+    """
+    top_doc = doc_hierarchy_with_zones.top
+    bottom_doc = doc_hierarchy_with_zones.bottom
+    middle_top_doc = doc_hierarchy_with_zones.middle_top
+    fr_top_doc = top_doc.translations.filter(locale='fr').first()
```

Comment:
You can say this shorter as ``fr_top_doc = top_doc.translations.get(locale='fr')``.